src/dotnet/commands/dotnet-compile-fsc/Program.cs

```diff
@@ -137,6 +134,13 @@ public static int Run(string[] args)
                 File.Move(outputName, originalOutputName);
             }
 
+            //HACK dotnet build require a pdb (crash without), fsc atm cant generate a portable pdb, so an empty pdb is created
```

Comment:
@davidfowl initially i have done like that, as you suggested in the issue.
I ignored copy if pdb file doesnt exists, but i think it's an error if pdb doesnt exists (it's expected to exists, only f# doesnt create it atm). 
The modify was ignoring all not existing files (ignore only pdb is an hack).
Reporting an error and abort it's useless for me, i want to make f# work.
I tried to add a warning (instead of error) but the code diff was too large, and change of merge is low.

Its already a bit hard for me merge changes in `dotnet-compile-fsc` alone, so i was trying to workaround in `dotnet-compile-fsc` code without changing the rest of code in this repo. 
Maybe like that it's faster to merge (and for sure i dont want introduce bugs in c#).